package.json

```diff
@@ -12,6 +12,7 @@
     "next": "14.2.4",
     "react": "^18",
     "react-dom": "^18",
+    "styled": "^1.0.0",
```

Comment:
What dependency is this? Are you using on your project? if not uninstall it, shceisse